Week3/homework/4-step3.js

```diff
@@ -1,2 +1,14 @@
 'use strict';
-// paste your freeCodeCamp solutions in here
+function Dog() {
+  this.name = "Rupert";
+  this.color = "brown";
+  this.numLegs = 4;
+}
+
+let hound = new Dog();
+////////////////
+function Dog() {
```

Comment:
This is a redefinition of the `Dog()` constructor function. You should never redefine a function in the same scope. At minimum it is a bad practice; usually it is considered an error.

So what value do you think  the` hound` variable has? Is it a`Rupert` or  a `Jacky`?